packages/nextjs/utils/scaffold-eth/notification.tsx

```diff
@@ -9,20 +9,20 @@ import {
 } from "@heroicons/react/24/solid";
 import { Spinner } from "~~/components/assets/Spinner";
 
-type TPositions = "top-left" | "top-center" | "top-right" | "bottom-left" | "bottom-center" | "bottom-right";
+type NotificationPositions = "top-left" | "top-center" | "top-right" | "bottom-left" | "bottom-center" | "bottom-right";
```

Comment:
I think we can remove this type and use the type `ToastPosition` from `react-hot-toast` 🙌